var/spack/repos/builtin/packages/memsurfer/package.py

```diff
@@ -12,32 +12,31 @@ class Memsurfer(PythonPackage):
        wide variety of large-scale molecular simulations."""
 
     homepage = "https://github.com/LLNL/MemSurfer"
-    git      = "git@github.com:LLNL/MemSurfer.git"
-    # url      = "https://github.com/LLNL/MemSurfer/archive/1.0.tar.gz"
+    git = "https://github.com/LLNL/MemSurfer.git"
 
```

Comment:
If you want to be notified of any build errors or PRs relating to this package, you can add:
```python
maintainers = ['bhatiaharsh']
```

Reply:
just made another commit with your suggestions